Example/Tests/Doubles/BitmovinPlayerTestDouble.swift

```diff
@@ -251,6 +275,47 @@ class BitmovinPlayerTestDouble: BitmovinPlayerStub, TestDoubleDataSource {
     }
 }
 
+class TestAdBreak: NSObject, AdBreak {
+    var identifier: String
+    var scheduleTime: TimeInterval
+    var ads: [any Ad]
```

Comment:
This `any` should not be needed. Ad is not a generic type. **applies for all**

```suggestion
    var ads: [Ad]
```

Reply:
https://github.com/bitmovin/bitmovin-player-ios-analytics-conviva/pull/68/commits/9a465cd6a083bf712e1661d08117504f48d5933b